### PR TITLE
Fix coco dataset type in MLIR

### PR DESCRIPTION
**Pull request by jackalcooper**
### reviewers:
@chengtbf : 
- oneflow/core/graph/op_graph.cpp
- oneflow/core/job_rewriter/insert_nccl_logical_op_pass.cpp

@liujuncheng oneflow/core/job_rewriter/cudnn_fused_normalization_add_relu_pass.cpp

### CODE REVIEW

**Review by chengtbf — APPROVED** [review]

File: oneflow/ir/oneflow-translate/lib/OneFlow/MLIROneFlowTranslation.cpp [hunk 1]
```diff
@@ -175,10 +176,12 @@ LogicalResult JobImporter::AddDeviceName(const ::oneflow::OperatorConf& op,
...
   Type ret = this->GetBuilder().getNoneType();
   job_wrapper_.QueryLogicalBlob(
-      lbn,
-      [this, &ret](const int64_t* shape_begin, const int64_t* shape_end, ::oneflow::DataType dt) {
+      lbn, [this, &ret, &lbn](const int64_t* shape_begin, const int64_t* shape_end,
+                              ::oneflow::DataType dt) {
         if (auto t = this->GetTypeFromOneFlowDataType(dt)) {
           ret = RankedTensorType::get(ArrayRef<int64_t>(shape_begin, shape_end), t.getValue());
+        } else {
+          llvm::errs() << "fail to get data type for: " << lbn << "\n";
```
Comment (BBuf) [inline oneflow/ir/oneflow-translate/lib/OneFlow/MLIROneFlowTranslation.cpp:184]:
```suggestion
          llvm::errs() << "fail to get data tensor type for: " << lbn << "\n";
```

File: oneflow/ir/oneflow-translate/lib/OneFlow/Importer.cpp [hunk 2]
```diff
@@ -394,6 +368,24 @@ Attribute ConvertNdSbpToAttr(Builder& builder, const ::oneflow::NdSbp& nd_sbp) {
   return builder.getStrArrayAttr(makeArrayRef(sbp_strrefs));
 }
 
+LogicalResult validateOpConf(const ::oneflow::OperatorConf& op_conf, UserOpArgs args,
```
Comment (BBuf) [inline oneflow/ir/oneflow-translate/lib/OneFlow/Importer.cpp:371]: 可以改名为
```suggestion
LogicalResult validateUserOpConf(const ::oneflow::OperatorConf& op_conf, UserOpArgs args,
```
这样？

**Review by BBuf — APPROVED** [review]

**Comment by jackalcooper** [comment]
@Mergifyio refresh

**Comment by jackalcooper** [comment]
@Mergifyio refresh

**Pull request merged**